cognito-pre-token-generation-openid/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     implementation libs.nva.auth
     implementation libs.aws.lambda.events
     implementation libs.zalando.problem
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.9.0'
```

Comment:
gson does not seem to being used and if you add dependencies follow the current style